sdk/clientcore/annotation-processor/src/main/java/io/clientcore/annotation/processor/templating/JavaParserTemplateProcessor.java

```diff
@@ -101,25 +98,18 @@ public void process(TemplateInput templateInput, ProcessingEnvironment processin
         String serviceInterfaceShortName = templateInput.getServiceInterfaceShortName();
 
         addImports(templateInput);
-        addOrphanComments();
+        addCopyrightComments();
         setPackageDeclaration(packageName);
+        createClass(serviceInterfaceImplShortName, serviceInterfaceShortName, templateInput, processingEnv);
 
-        createClass(serviceInterfaceImplShortName, serviceInterfaceShortName, templateInput);
-        try (Writer fileWriter = processingEnv.getFiler()
-            .createSourceFile(packageName + "." + serviceInterfaceImplShortName)
-            .openWriter()) {
-            fileWriter.write(compilationUnit.toString());
-            fileWriter.flush();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
+        writeFile(packageName, serviceInterfaceImplShortName, processingEnv);
     }
 
     void addImports(TemplateInput templateInput) {
         templateInput.getImports().keySet().forEach(compilationUnit::addImport);
     }
 
-    void addOrphanComments() {
+    void addCopyrightComments() {
         compilationUnit.addOrphanComment(new LineComment(" Copyright (c) Microsoft Corporation. All rights reserved."));
         compilationUnit.addOrphanComment(new LineComment(" Licensed under the MIT License."));
```

Comment:
Theoretically anyone can use the clientcore annotation-processor, and it might not be Microsoft. Eventually we'll need to find a way to externalise this string